### DISCUSSION TITLE
Withdrawal fund's

### DISCUSSION THREAD

**Discussion by Jpspartan** [btcpayserver/btcpayserver#5663] [Ideas / Feature Requests]
Can I have some moral support for the Bitcoin mining funds funds please?

**Comment by dennisreimann** [comment]
Beware of scams, see https://blog.btcpayserver.org/bitcoin-bonus-scam/

**Reply by bladewatcher** [reply]
thanks Dennis I've been beating myself up and  down in and out over this ,,Ive  gotten about 10 of those ..maybe more ..ive picked them apart and wishing they was real.. Tonight i was getting ready to pay my 64.oo just maybe just once maybe its a real thing. Coz ive been almost scammed by hundreds and only once gave in to the sugar mamma scam..lol  Im still giving her grief  .. but I still got them thinking I'm paying so If I got any thing you think I might have let me know.. ok buddy thanks again  george blade  
ps i took a snapshot of this . oops ,, i got it its the a pic of the security  cert.. .
